### DISCUSSION TITLE
記事カード（コメントバージョン）に記事のいいね数とコメントのいいねアイコンが混在

### DISCUSSION THREAD

**Discussion by aotoriii** [increments/qiita-discussions#324] [要望・アイデア・不具合]
タイムラインやコメントした記事の記事カード（コメントバージョン）についてです。
下部に表示されるいいね数はコメントのではなく、記事のいいね数です。
一方、左横のいいねアイコンの色は記事のではなく、コメントのいいねによります。

#215 の回答により、「コメントした記事のLGTM数」（今はいいね数）になっているのは仕様だと認識しております。
いいねボタンの色については、仕様か不具合か判断しかねますが、
私にとっては分かりづらくて、混乱しました。
コメントのみをいいねした場合、記事にいいねした覚えがないのに、今のUIだといいねしたかのように受け取ってしまいます。

#### 記事のみをいいねした時
![lgtm-1](https://user-images.githubusercontent.com/73811757/195743036-5f7542b1-3773-40f2-bc16-cdbee0103160.png)

#### コメントのみをいいねした時
![lgtm-2](https://user-images.githubusercontent.com/73811757/195743038-152f6c28-16e0-41aa-9a32-67b16a374188.png)

仕様であれば、いいね数とアイコンを記事かコメントかのどちらかに合わせてほしいです。
ご検討いただけますと幸いです。

**Comment by tanaka-asami415** [comment]
@aotoriii 
ご報告いただきありがとうございます！
不具合なので修正します🙏

**Comment by mayobimu** [comment]
@aotoriii 
お待たせいたしました。
こちら対応完了しました！
